[front] - fix(ConversationSidePanelContainer): hide resizable handle on small screens

JulesBelveze opened the pull request:
## Description

Make the resizable handle for panels hidden on small screens and only visible on medium and larger devices

## Tests

Locally, opening a breakdown panel

## Risk

Low

## Deploy Plan

Deploy front

Fraggle commented on front/components/assistant/conversation/ConversationSidePanelContainer.tsx:
```diff
@@ -38,7 +38,9 @@ export default function ConversationSidePanelContainer({
       {/* Resizable Handle for Panels */}
-      {currentPanel && <ResizableHandle withHandle className="z-50" />}
+      {currentPanel && (
+        <ResizableHandle withHandle className="z-50 hidden md:flex" />
```
I have a different fix using "useIsMobile()", wdyt is the most explicit ? (At least, your change requires a comment imho)

JulesBelveze committed:
[front/assistant/conversation] - fix: adjust resizable panel handle visibility on mobile

 - Hides the resizable handle for conversation panels when on mobile devices to improve usability
 - Utilizes the `useIsMobile` hook to determine if the application is being used on a mobile device

Fraggle reviewed (approved):
Approved
